package/Dockerfile

```diff
@@ -19,22 +19,28 @@
 # Note that fluentd is run with root permssion to allow access to
 # log files with root only access under /var/log/containers/*
 
-FROM alpine:3.14
+FROM debian:stable-slim
 
+ARG DEBIAN_FRONTEND=noninteractive
+
+COPY clean-apt /usr/bin
+COPY clean-install /usr/bin
 COPY Gemfile /Gemfile
 COPY Gemfile.lock /Gemfile.lock
 
-RUN apk update \
-    && apk add --no-cache ca-certificates ruby ruby-dev ruby-irb ruby-etc ruby-webrick build-base \
+RUN BUILD_DEPS="make ruby-dev build-essential autoconf automake libtool libsnappy-dev" \
+    && clean-install $BUILD_DEPS \
+                     ca-certificates \
+                     ruby \
     && echo 'gem: --no-document' >> /etc/gemrc \
     && gem install bundler \
-    && bundle install \
+    && CFLAGS=-Wno-error=format-overflow bundle install \
+    && apt-get purge -y --auto-remove \
+                     -o APT::AutoRemove::RecommendsImportant=false \
+                     $BUILD_DEPS \
+    && clean-apt \
     # Ensure fluent has enough file descriptors
-    && ulimit -n 65536 \
-    && apk del build-base \
-    && rm -rf /tmp/* /var/tmp/* /usr/lib/ruby/gems/*/cache/*.gem /usr/lib/ruby/gems/2.*/gems/fluentd-*/test \
-    # Remove file due to red herring: https://github.com/fluent/fluentd/issues/3374#issuecomment-840916184
-    && rm -f /usr/lib/ruby/gems/2.7.0/gems/http_parser.rb-0.6.0/Gemfile.lock
+    && ulimit -n 65536
 
 # Copy the Fluentd configuration file for logging Docker container logs.
 COPY fluent.conf /etc/fluent/fluent.conf
```

Comment:
Looks like not everything from original move is here, is that on purpose ? https://github.com/rancher/fluentd/pull/42/files#diff-c55d1ee74f32cd4b9eeeceaec981a180f52e761acb158a5b0b0e088c7725c9caL59

Reply:
Do these bash scripts not need to update either ? https://github.com/rancher/fluentd/pull/42/files#diff-d0dab9599d947b7177a6ef7453f6ccc55151b7c70d52ad2ada1ba78caa438093R23

Reply:
This was merged into newer Debian releases. We are good here: https://github.com/rancher/fluentd/pull/51#discussion_r698774010

Reply:
Echoing the same @paynejacob : https://github.com/rancher/fluentd/pull/51#discussion_r698774631

Reply:
The `jmalloc1` omission was intentionally, ill add back the cleanup commands and run change